tests/unit/core/utils/test_compatibility.js

```diff
@@ -165,56 +162,6 @@ describe(__filename, () => {
       ).toEqual({ compatible: false, reason: INCOMPATIBLE_FIREFOX_FOR_IOS });
     });
 
-    it('should mark Firefox without window.external as incompatible', () => {
-      const userAgentInfo = {
-        browser: { name: 'Firefox' },
-        os: { name: 'Windows' },
-      };
-      const fakeOpenSearchAddon = createInternalAddon({
-        ...fakeAddon,
-        type: ADDON_TYPE_OPENSEARCH,
-      });
-      const fakeWindow = {};
-
-      expect(
-        _isCompatibleWithUserAgent({
-          _window: fakeWindow,
-          addon: fakeOpenSearchAddon,
-          userAgentInfo,
-        }),
-      ).toEqual({ compatible: false, reason: INCOMPATIBLE_NO_OPENSEARCH });
-    });
-
-    it('should mark Firefox without OpenSearch support as incompatible', () => {
-      const fakeOpenSearchAddon = createInternalAddon({
-        ...fakeAddon,
-        type: ADDON_TYPE_OPENSEARCH,
-      });
-      const fakeWindow = { external: {} };
-
-      expect(
-        _isCompatibleWithUserAgent({
-          _window: fakeWindow,
-          addon: fakeOpenSearchAddon,
-        }),
-      ).toEqual({ compatible: false, reason: INCOMPATIBLE_NO_OPENSEARCH });
-    });
-
-    it('should mark Firefox with OpenSearch support as compatible', () => {
-      const fakeOpenSearchAddon = createInternalAddon({
-        ...fakeAddon,
-        type: ADDON_TYPE_OPENSEARCH,
-      });
-      const fakeWindow = createFakeMozWindow();
```

Comment:
We can also remove this `createFakeMozWindow()` function now. I don't think it is being used anymore.

Reply:
Cool. Removed it from `tests/unit/helpers.js`